packages/theme-chalk/src/input-number.scss

```
@@ -11,6 +11,47 @@
   line-height: #{map.get($input-height, 'default') - 2};
   vertical-align: middle;
 
+  &::after {
+    content: '';
+    position: absolute;
+    inset: 0;
+    z-index: 2;
+    pointer-events: none;
+    border-radius: getCssVar('border-radius', 'base');
```

Stylelint (17.14.0)

[error] Expected "getCssVar" to be "getcssvar" (function-name-case)

(function-name-case)